mud-contracts/world/src/modules/inventory/systems/InventoryInteract.sol

```diff
@@ -84,6 +84,64 @@ contract InventoryInteract is EveSystem {
     _inventoryLib().depositToEphemeralInventory(smartObjectId, ephemeralInventoryOwner, inItems);
   }
 
+  /**
+   * @notice Transfer items from ephemeral to inventory
+   * @dev transfer items from ephemeral to inventory
+   * @param smartObjectId is the smart object id
+   * @param ephemeralInventoryOwner is the ephemeral inventory owner
+   * @param outItems is the array of items to transfer
+   */
+  function inventoryToEphemeralTransferWithParam(
+    uint256 smartObjectId,
+    address ephemeralInventoryOwner,
```

Comment:
For interfacing this is clearer, but technically we can just use the parameter in the InventoryItem object (as now it is not being used). In any case, I am fine to leave it as is for now.